db/migrate/20241030000131_add_post_header_to_badge.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class AddPostHeaderToBadge < ActiveRecord::Migration[7.1]
+  def change
+    add_column :badges, :post_header, :boolean, default: false, null: false
```

Comment:
```suggestion
    add_column :badges, :show_in_post_header, :boolean, default: false, null: false
```

Maybe something like the suggestion above for the column name? It is more verbose but I find it much clearer in indicating what the column is used for.